src/library_memfs.js

```diff
@@ -205,6 +205,10 @@ addToLibrary({
             }
           }
         }
+        try {
+          new_node = FS.lookupNode(new_dir, new_name);
+          FS.hashRemoveNode(new_node);
+        } catch (e) {}
```

Comment:
Can do just do this the try/catch once above?
e.g.

```
var new_node;
try {
   new_node = FS.lookupNode(new_dir, new_name);
} catch (e) {}

if (new_node) {
   if (FS.isDir(old_node.mode)) {
     ...
   }
   FS.hashRemoveNode(new_node);
}
```

Reply:
Done.